en/02_Developer_Guides/15_Customising_the_Admin_Interface/04_Preview.md

```diff
@@ -405,7 +383,7 @@ class MyAdmin extends ModelAdmin
 >         // Render the preview content
 >         $preview = $obj->forTemplate();
 >         // Wrap preview in proper html, body, etc so Requirements are used
->         $preview = SSViewer::create('PreviewBase')->process(ArrayData::create(['Preview' => $preview]));
+>         $preview = SSViewer::create('PreviewBase')->process(['Preview' => $preview]);
```

Comment:
No need to wrap them anymore. I'd argue it's even best practice to not wrap them.